crates/image_viewer/src/image_info.rs

```diff
@@ -0,0 +1,113 @@
+use crate::ImageView;
+use gpui::{div, AppContext, IntoElement, ParentElement, Render, Subscription, View, ViewContext};
+use settings::{ImageFileSizeUnitType, Settings};
+use ui::{prelude::*, Button, LabelSize};
+use workspace::{ItemHandle, StatusItemView, Workspace};
+
+pub struct ImageInfo {
+    width: Option<u32>,
+    height: Option<u32>,
+    file_size: Option<u64>,
+    color_type: Option<String>,
```

Comment:
As mentioned below, these 4 values should have their own named struct. Then you can just have a single optional field for them all :)